### PR TITLE
improve tracy docs

### PR DESCRIPTION
**Pull request by bjacob**

### CODE REVIEW

File: docs/developing_iree/profiling_with_tracy.md [hunk 1]
```diff
@@ -162,42 +164,81 @@ If connecting doesn't work:
...
+section. Look for any interesting `stderr` message (in the profiled program's
+terminal).
+
+Click the 'Statistics' button at the top. It will open a window like this:
+
+![Tracy statistics
+window](https://gist.githubusercontent.com/bjacob/ff7dec20c1dfc7d0fc556cc7275bca9a/raw/fe4e22ca0301ebbfd537c47332a4a2c300a417b3/tracy_statistics.jpeg)
+
+See how the above screenshot has two radio buttons at the top: 'Instrumentation'
+and 'Sampling'. At this point, if you don't a the 'Sampling' radio button, you
```
Comment (ScottTodd) [inline docs/developing_iree/profiling_with_tracy.md:196]: typo?

```suggestion
and 'Sampling'. At this point, if you don't see the 'Sampling' radio button, you
```

File: docs/developing_iree/profiling_with_tracy.md [hunk 2]
```diff
@@ -162,42 +164,81 @@ If connecting doesn't work:
...
+data that Tracy collects about your program. In the Tracy main view, they
+correspond, respectively, to 'normal' and 'ghost' zones. For each thread, the
+ghost icon toggles the view between these two kinds of zones. 'Normal' zones are
+explicitly recorded by the instrumented program, with timestamps. The stacks
+shown are stacks of Tracy instrumentation zones, not real call stacks. 'Ghost'
+zones are obtained by Tracy sampling the thread's real call stack at a frequency
+of typically roughly 10 kHz. Only ghost zones have the data to show the time
+spent on a line-by-line or instruction-by-instruction basis. That's why when we
+actually click on a zone, we get a very different view depending on whether it's
+a normal or a ghost zone.
```
Comment (ScottTodd) [inline docs/developing_iree/profiling_with_tracy.md:210]: If this gets more comprehensive, I'd suggest referring people to the Tracy manual for a general introduction to Tracy and then focusing our docs on what is noteworthy about IREE (e.g. which threads and event patterns to look for).

**Review by ScottTodd — APPROVED** [review]

File: docs/developing_iree/profiling_with_tracy.md (hunk 1 above)
Comment (bjacob) [reply docs/developing_iree/profiling_with_tracy.md:196]: thanks!

File: docs/developing_iree/profiling_with_tracy.md (hunk 2 above)
Comment (bjacob) [reply docs/developing_iree/profiling_with_tracy.md:210]: good point, linked and trimmed this text.

**Pull request merged**